Throttler.pm

```diff
@@ -478,7 +481,14 @@ sub try_push {
     }
 
     if($val >= $self->{max_items}) {
-        DEBUG "Not increasing counter $key by $count (already at max)";
+        if ($force) {
+            DEBUG "Increasing (force) counter $key by $count ",
+                  "($val|$self->{max_items})";
+            $b->{count}->{$key} += $count;
+        }
+        else {
```

Comment:
Minor nitpick: please use the same brace/indentation style:

 } else {
        #...
 }